### PR TITLE
(feat) Updated code lines Added

### PR DESCRIPTION
**Pull request by neeraj542**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction). 
- [ ] My work includes tests or is validated by existing tests. 

## Summary

Implementing codebase and making new changes with new modules and components. Updating as following lines:

- Created a new state variable to store the current sort column and direction and Update the getHeaderProps function to handle column sorting. ([O3-904](https://issues.openmrs.org/browse/O3-904))
- Import the correct launchFormEntry function from the @openmrs/esm-form-entry module. ([View](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1095/commits/da578a9fa3b718a5933221c73506035ca1d18648))
- Updated code is now syntactically correct and follows best practices for importing and using dependencies in Sass stylesheets.

## Test
- function updation to column handling
- Importing correct function and replaced by unusuals
- Also following the best practices for import dependencies.

## Screenshots

image

### CODE REVIEW

**Review by Juliet-chime — APPROVED** [review]
LGTM

**Review by Juliet-chime — COMMENTED** [review]
Please add Test scenario for this implementation

File: packages/esm-patient-biometrics-app/src/biometrics/paginated-biometrics.component.tsx
Comment (neeraj542) [inline packages/esm-patient-biometrics-app/src/biometrics/paginated-biometrics.component.tsx:1]: Created a new state variable to store the current sort column and direction and Update the getHeaderProps function to handle column sorting.

**Comment by brandones** [comment]
Hi @neeraj542 . I cannot make sense of what you have done. This looks like several unrelated changes all in the same PR. I don't think you've taken the right approach to the sorting-pagination problem. I am going to close this.

If you want to contribute in the future, please ensure that your PR corresponds to exactly one ticket, that it fixes the problem specified in that ticket, and that the screenshots or screen recordings make it clear that you have indeed fixed the problem.

**Pull request closed without merging**